pulsar-client-cpp/tests/UrlTest.cc

```diff
@@ -16,68 +16,76 @@
+  ASSERT_EQ("example.com", url.host());
+  ASSERT_EQ("http", url.protocol());
+  ASSERT_EQ(8080, url.port());
 
-    ASSERT_TRUE(Url::parse("http://example.com:8080/test/my/path?key=value#adsasda", url));
-    ASSERT_EQ("example.com", url.host());
-    ASSERT_EQ("http", url.protocol());
-    ASSERT_EQ(8080, url.port());
+  ASSERT_TRUE(Url::parse("http://example.com:8080/test/my/path?key=value#adsasda", url));
+  ASSERT_EQ("example.com", url.host());
+  ASSERT_EQ("http", url.protocol());
+  ASSERT_EQ(8080, url.port());
 
-    ASSERT_TRUE(Url::parse("pulsar://example.com:8080", url));
-    ASSERT_EQ("example.com", url.host());
-    ASSERT_EQ("pulsar", url.protocol());
-    ASSERT_EQ(8080, url.port());
+  ASSERT_TRUE(Url::parse("pulsar://example.com:8080", url));
+  ASSERT_EQ("example.com", url.host());
+  ASSERT_EQ("pulsar", url.protocol());
+  ASSERT_EQ(8080, url.port());
 
-    ASSERT_TRUE(Url::parse("pulsar://example.com", url));
-    ASSERT_EQ("example.com", url.host());
-    ASSERT_EQ("pulsar", url.protocol());
-    ASSERT_EQ(6650, url.port());
+  ASSERT_TRUE(Url::parse("pulsar://example.com", url));
+  ASSERT_EQ("example.com", url.host());
+  ASSERT_EQ("pulsar", url.protocol());
+  ASSERT_EQ(6650, url.port());
 
-    ASSERT_TRUE(Url::parse("http://env-broker3.messaging.cluster.company.com:4080/lookup/v2/destination/persistent/cmscpp/gq1/TESTNS.4/TOPIC_1490664894335_1?authoritative=false", url));
-    ASSERT_EQ("http", url.protocol());
-    ASSERT_EQ(4080, url.port());
-    ASSERT_EQ("/lookup/v2/destination/persistent/cmscpp/gq1/TESTNS.4/TOPIC_1490664894335_1", url.path());
-    ASSERT_EQ("/lookup/v2/destination/persistent/cmscpp/gq1/TESTNS.4/", url.pathWithoutFile());
-    ASSERT_EQ("TOPIC_1490664894335_1", url.file());
-    ASSERT_EQ("?authoritative=false", url.parameter());
+  ASSERT_TRUE(Url::parse(
+      "http://env-broker3.messaging.cluster.company.com:4080/lookup/v2/destination/"
+      "persistent/cmscpp/gq1/TESTNS.4/TOPIC_1490664894335_1?authoritative=false",
+      url));
+  ASSERT_EQ("http", url.protocol());
+  ASSERT_EQ(4080, url.port());
+  ASSERT_EQ("/lookup/v2/destination/persistent/cmscpp/gq1/TESTNS.4/TOPIC_1490664894335_1",
+            url.path());
+  ASSERT_EQ("/lookup/v2/destination/persistent/cmscpp/gq1/TESTNS.4/",
+            url.pathWithoutFile());
+  ASSERT_EQ("TOPIC_1490664894335_1", url.file());
+  ASSERT_EQ("?authoritative=false", url.parameter());
 
-    ASSERT_TRUE(Url::parse("http://abc.com:8090/ads/ad/asd/TOPIC_1490664894335_1?authoritative=false,temp=true", url));
-    ASSERT_EQ("http", url.protocol());
-    ASSERT_EQ(8090, url.port());
-    ASSERT_EQ("/ads/ad/asd/TOPIC_1490664894335_1", url.path());
-    ASSERT_EQ("/ads/ad/asd/", url.pathWithoutFile());
-    ASSERT_EQ("TOPIC_1490664894335_1", url.file());
-    ASSERT_EQ("?authoritative=false,temp=true", url.parameter());
+  ASSERT_TRUE(
+      Url::parse("http://abc.com:8090/ads/ad/asd/"
```

Comment:
Is there a way to ignore lines from being formatted.

Reply:
```
// clang-format off
...
// clang-format on
```
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code